blamo/lib/main.dart

```diff
@@ -1,142 +1,231 @@
 import 'package:flutter/material.dart';
+import 'package:blamo/Export/index.dart';
 
-void main() => runApp(MyApp());
+//This class will be used to house all the data between each route
+class StateData {
+  String currentRoute;
+  int randomNumber;
+  var list = ["Document 1","Document 2","Document 3","Document 4","Document 5","Document 6"];
+
+  StateData(this.currentRoute, [this.randomNumber = 6]);
+}
+
+/* the idea behind the home page is a series of existing logs will appear in the white space, While the button in
+  * the bottom right will allow users to create a new log
+  * (the "second page" in this code is mostly a demonstration and can/should be removed in later implimentation) additionally,
+  * the drawer will provide easy familiar navigation between setting, export, etc. Activities/pages of the project
+  */
+void main() {
+  runApp(new MaterialApp(
+      home: HomePage(),
+      routes: <String, WidgetBuilder> {
+        '/SecondPage': (BuildContext context) => new SecondPage(),
+        '/ExportPage': (BuildContext context) => new ExportPage()
```

Comment:
Build error here, export files missing. Wait for Evan to handle in his PR or leave in?